.buildkite/scripts/steps/dra-publish.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -uo pipefail
+
+source .buildkite/scripts/bootstrap.sh
+
+# Publish DRA artifacts
+function run_release_manager() {
+    echo "+++ Publishing $BUILDKITE_BRANCH ${WORKFLOW} DRA artifacts..."
+    dry_run=""
+    if [ "$BUILDKITE_PULL_REQUEST" != "false" ]; then
+        dry_run="--dry-run"
+        # force main branch on PR's or it won't execute
+        # because the PR branch does not have a project folder in release-manager
+        BRANCH=main
+    fi
+    docker run --rm \
+        --name release-manager \
+        -e VAULT_ADDR="${VAULT_ADDR_SECRET}" \
+        -e VAULT_ROLE_ID="${VAULT_ROLE_ID_SECRET}" \
+        -e VAULT_SECRET_ID="${VAULT_SECRET}" \
+        --mount type=bind,readonly=false,src="${PWD}",target=/artifacts \
+        docker.elastic.co/infra/release-manager:latest \
```

Comment:
Using the `latest` tag is a suggestion from release engineering? Is it guaranteed to point to the latest *stable* version of the image ?

Reply:
Yes, this is how we call the release-manager.